src/Features/LanguageServer/Protocol/Protocol/TextDocumentChangeRegistrationOptions.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
 using System.Runtime.Serialization;
```

Comment:
```suggestion
```

might be able to remove some of the other usings too

Reply:
So used to this being an error in razor 😅